Decred Wallet/Custom Views/Banner.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Banner.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+
+enum BannerType: String {
+    case success
+    case error
+}
+
+class Banner: UIView {
+    private var parent:UIViewController?
+    
+    init(parent: UIViewController) {
+        super.init(frame: .zero)
+        self.parent = parent
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    public func show(type: BannerType, text:String) {
+        guard let parent = self.parent else { return }
+        if self.superview == parent.view { return }
+
+        parent.view.addSubview(self)
+        let guide = parent.view.safeAreaLayoutGuide
```

Comment:
`guide` is only used once below, no need for this declaration, can just use `parent.view.safeAreaLayoutGuide` directly.